packages/bpk-component-badge/readme.md

```diff
@@ -14,15 +14,14 @@ npm install bpk-component-badge --save-dev
 import React from 'react';
 import BpkBadge from 'bpk-component-badge';
```

Comment:
As we're using `BADGE_TYPES` in the prop types table, it should be imported and used in the example code too.